languages/csharp/AppEncryption/scripts/install_local.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e
+
+FULL_VERSION=$(xmllint --xpath "//Project/PropertyGroup/Version/text()" Directory.Build.props)
+TIME_STAMP=$(date "+%Y%m%d")
```

Comment:
we should have more than just the date. be good to match the resolution the maven snapshot builds go down to (seconds)

Reply:
Can we just use the epoch time?

Reply:
i think that's less intuitive when looking at it (especially if we do ever decide to deploy these down the road). should just be a matter of expanding the format you're specifying